include/smac_planner/utils.hpp

```diff
@@ -82,6 +83,23 @@ class Utils {
     }
 
 
+  /**
+  * @brief Check if the input poses are within specified tolerance
+  * @return Bool true they are under tolerance, false means they are not in tolerance
+  */
+  static inline bool arePoseWithinTolerance(geometry_msgs::Pose pose1, geometry_msgs::Pose pose2, double tolerance){
+
+    double yaw_offset = std::abs(tf2::getYaw(pose1.orientation) -  tf2::getYaw(pose2.orientation));
```

Comment:
F: need to use `angles::shortest_angular_distance`
if yaw = pi and other_yaw = -pi, they are the same
F: `const`